app/controllers/concerns/api/v2/lookup_keys_common_controller.rb

```diff
@@ -144,4 +122,11 @@ def return_if_smart_mismatch
       not_found "#{obj} not found by id '#{id}'"
     end
   end
+
+  private
+
+  def model_not_found(model_name)
+    not_found(:error => { :message => (_("#{model_name.capitalize} with id '%{id}' was not found") %
```

Comment:
Also since you're doing this programmatically you'll need to avoid .capitalise and do something like `_(model_name)` but with the `model.name` variable, not the lower case version.  (See locale/model_attributes.rb for the ones we translate.)